src/platforms/ruby/index.mdx

```diff
@@ -4,6 +4,7 @@ caseStyle: snake_case
 supportLevel: production
 redirect_from:
   - /clients/ruby/
+  - /platforms/ruby/config/
```

Comment:
I think we may want to redirect to https://docs.sentry.io/platforms/ruby/configuration/ instead of the index page (same for others)